cmd/signature-v4_test.go

```diff
@@ -54,14 +54,7 @@ func TestDoesPolicySignatureMatch(t *testing.T) {
 			},
 			expected: ErrInvalidAccessKeyID,
 		},
-		// (2) It should fail if the region is invalid.
-		{
-			form: http.Header{
-				"X-Amz-Credential": []string{fmt.Sprintf(credentialTemplate, accessKey, now.Format(yyyymmdd), "invalidregion")},
```

Comment:
Yeah it would fail with signature does not match. since we allowed the region.. let's say region is not set on server and client sets it.. Then this error case shouldn't fail. So we need to figure out why it is reporting as such.